boms/cloud-oss-bom/pom.xml

```diff
@@ -267,7 +268,46 @@
          <artifactId>grpc-google-iam-v1</artifactId>
          <version>${iam.protos.version}</version>
        </dependency>
-
+       <dependency>
+         <groupId>com.google.cloud.sql</groupId>
```

Comment:
Does this project have its own BOM? Should i?

Reply:
No, it doesn't have a BOM. I can ask them to release one.